src/ceph-volume/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py35, py36, flake8
```

Comment:
We backport all the things to nautilus and mimic. Having no py27 checks will make backporting harder for no reason?

Reply:
> Having no py27 checks will make backporting harder

i disagree. every backport PR runs through "make check" run. if it's harder, probably yes. the only reason it's harder is that a change is not tested with python2.7 at the first place when it is posted when it's targeting master.

also, i think it's a compromise. the problem of "will make backporting harder" always exist. and it actually applies to all changes: a backport is harder to test when it is backported.

> for no reason?

i don't agree with this part. the reason to remove py27 from envlist is

1. to avoid the headache of developer working on "python3 distros"
2. there is no reason to test against python2 if we don't support it in octopus anymore.

Reply:
I'm just imagining some roundtrips coming. E.g. 

* New PR for mater ceph-volume. 
* PR gets merged into master.
* Backport PR discovers that there is a problem with py2
* New PR for mater ceph-volume to address the py2 problem
* Merge this PR and cherry-pick he fix into the original backport PR. 

By continue to test py2, we can avoid this?

Reply:
> New PR for mater ceph-volume to address the py2 problem

how come we need to fix an issue on master which does not exist on master?

Reply:
I lean towards shedding as much of the legacy/overhead on master as possible (i.e., dropping py2 from master).

Reply:
My take on this would be to keep py2, until backporting to py2 becomes the exception.

Reply:
> I lean towards shedding as much of the legacy/overhead on master as possible (i.e., dropping py2 from master).

While I agree with this, I'd also prefer to keep the py2 in ceph-volumes tox.ini. This is certainly not ideal but due to $past_choices ceph-volume is the oddball in the ceph repo. @sebastian-philipp listed the advantages of this.
And the tests for ceph-volume will keep working in py2 and 3, there is no dependency on `install-deps.sh`, so no harm done.

Reply:
@jan--f i am dropping this change. and will defer the decision to you and @alfredodeza . i cannot find the advantages listed by @sebastian-philipp. what he listed is a case, and i don't think it makes sense.

Reply:
Thx. The advantage is that ceph-volume needs to be py2 and 3 compatible since we essentially maintain the same code stream (or close enough) in all current release branches. So its easier if we can check py2 compatibility on master.